app/controllers/user.js

```diff
@@ -1,33 +1,112 @@
 const User = require('../models').users,
-  logger = require('../logger');
+  errors = require('../errors'),
+  logger = require('../logger'),
+  jwt = require('jwt-simple'),
+  secret = process.env.NODE_API_JWT_SECRET_STRING,
```

Comment:
We actually have the `config/index.js` file to import all these secrets, so we have in one place all the env vars, and then if you wanna use that config value you just import the config file and use the key you want of the config object.
If I wans't clear I can explain little bit more later.